src/main/java/baseball/Computer.java

```diff
@@ -0,0 +1,65 @@
+package baseball;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Computer {
+
+    public int cntStrike = 0;
+    public int cntBall = 0;
+
+    public List<Integer> computer;
+
+    public List<Integer> selectNumbers() {
+        computer = new ArrayList<>();
+        while (computer.size() < Game.INPUT_NUMBER_LENGTH) {
+            int randomNumber = Randoms.pickNumberInRange(Game.START_NUMBER, Game.LAST_NUMBER);
+            if (!computer.contains(randomNumber)) {
+                computer.add(randomNumber);
+            }
+        }
+        return computer;
+
+    }
+
+    public void compare(String input) {
+        cntStrike = 0;
+        cntBall = 0;
+
+        for (int i = 0; i < computer.size(); i++) {
+            int num = Utils.stringToInt(input, i);
+            if ((num) == computer.get(i)) {
+                cntStrike += 1;
+            } else if (computer.contains(num)) {
+                cntBall += 1;
+            }
+
+        }
+    }
+
+    public void countResult() {
+        if (cntStrike == 0 && cntBall == 0) {
+            System.out.println("낫싱");
+        } else if (cntStrike == 0 && cntBall > 0 ) {
+            System.out.println(cntBall + "볼");
+        } else if (cntStrike > 0 && cntBall > 0) {
+            System.out.println(cntBall + "볼 "+ cntStrike + "스트라이크");
+        } else if (cntStrike > 0 && cntBall == 0 ) {
+            System.out.println(cntStrike + "스트라이크");
+            if (cntStrike ==Game.INPUT_NUMBER_LENGTH) {
+                System.out.println(Game.INPUT_NUMBER_LENGTH+"개의 숫자를 모두 맞히셨습니다! 게임 종료");
+            }
+        }
+
+    }
```

Comment:
```suggestion
    public void printResult() {
        ...
    }
```

개인적으로 countResult는 count한 result를 실제 반환값으로 반환하는 느낌이 듭니다.
대신 printResult를 제안드립니다.

추가로 "게임종료" 출력은 매 라운드의 결과표시와 묶이기 보다는, 게임 종료조건을 체크하는 아래 gameResult 메서드 내로 들어가는게 좋아보입니다.